docs/reference/setup/important-settings/gc-logging.asciidoc

```diff
@@ -1,7 +1,25 @@
 [[gc-logging]]
 === GC logging
 
-By default, Elasticsearch enables GC logs. These are configured in
-<<jvm-options,`jvm.options`>> and default to the same default location as the
-Elasticsearch logs. The default configuration rotates the logs every 64 MB and
-can consume up to 2 GB of disk space.
+By default, {es} enables GC logs. These are configured in
+<<jvm-options,`jvm.options`>> and default to the same default location as
+the Elasticsearch logs. The default configuration rotates the logs every 64
+MB and can consume up to 2 GB of disk space.
+
+You can change the default GC logging configuration. Since the preferred
+method of <<jvm-options,configuring JVM options>> is not to edit the
+default `jvm.options` file, you will need to configure the JVM first to
+disable GC logging, in order to cancel out the default {es} configuration,
+and then to re-enable logging as you require. See
+https://openjdk.java.net/jeps/158[JEP 158: Unified JVM Logging] for details
+of how to configure logging.
```

Comment:
I think a slight reorder and formatting would help here:

```
We do not recommend changing the default `jvm.options` file to change the GC
logging configuration. Instead, we recommend you:

. Configure the JVM to disable GC logging using the default configuration.
. Reconfigure JVM logging as described in
  https://openjdk.java.net/jeps/158[JEP 158: Unified JVM Logging].
. Re-enable GC logging.
```

It may also be helpful to briefly mention or link to:

1.  Why we discourage editing the `jvm.options` file.
2. How to disable and re-enable GC logging.